languagetool-language-modules/en/src/main/java/org/languagetool/language/English.java

```diff
@@ -539,6 +606,9 @@ protected int getPriorityForId(String id) {
       if (id.startsWith("AI_HYDRA_LEO_CP")) {
         return 2;
       }
+      if (id.startsWith("AI_HYDRA_LEO_CP_YOU")) {
+        return 1;
+      }
       if (id.startsWith("AI_HYDRA_LEO_MISSING_A")) {
```

Comment:
@palasso Is this okay?

Reply:
I haven't added that part, so I am not entirely sure of its purpose. Maybe @s-burst knows more for this.

As I see it, line 610 would never execute since `AI_HYDRA_LEO_CP_YOU` starts with `AI_HYDRA_LEO_CP`.

Either this part should be removed entirely both in open source and premium or be placed before 606-608.

Reply:
Thanks. Certainly, it does nothing as it is now.